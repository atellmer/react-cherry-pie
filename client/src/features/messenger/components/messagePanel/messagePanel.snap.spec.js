import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

import TmMessagePanel from './index';

const props = {
    changeMessagePanelHeight: () => {}
  };

test('Jest: TmMessagePanel (Snapshot)', () => {
  const shallowRenderer = ReactTestUtils.createRenderer();
  const tree = shallowRenderer.render(<TmMessagePanel {...props}/>);

  expect(tree).toMatchSnapshot();
});
