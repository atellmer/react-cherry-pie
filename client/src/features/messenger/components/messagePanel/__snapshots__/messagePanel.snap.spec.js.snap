// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest: TmMessagePanel (Snapshot) 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className={undefined}
  >
    <div>
      <IconButton
        disableTouchRipple={false}
        disabled={false}
        iconStyle={Object {}}
        tooltipPosition="bottom-center"
        touch={false}
      >
        <SocialSentimentSatisfied
          color="#7f7f7f"
        />
      </IconButton>
    </div>
    <div
      className={undefined}
    >
      <TmTextarea
        onChangeInput={[Function]}
        onHeightChange={[Function]}
        value=""
      />
    </div>
    <div>
      <IconButton
        disableTouchRipple={false}
        disabled={false}
        iconStyle={Object {}}
        tooltipPosition="bottom-center"
        touch={false}
        type="submit"
      >
        <ContentSend
          color="#9a85cd"
        />
      </IconButton>
    </div>
  </div>
</form>
`;
